ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/timeline/TimelineComponentView.kt

```diff
@@ -159,9 +181,9 @@ internal fun TimelineComponentView(
                             width = Dimension.value(item.connector?.width?.dp ?: 0.dp)
                             top.linkTo(currentIconRef.top)
                             if (isLastItem) {
-                                bottom.linkTo(parent.bottom)
+                                bottom.linkTo(parent.bottom, margin = offsets.second)
                             } else {
-                                bottom.linkTo(nextIconRef!!.top)
```

Comment:
This was wrong... I was relaying on the size of each icon being the same and the offset solving this solution, but if we support icons of different sizes, this doesn't work well.